recipes/qutip/meta.yaml

```diff
@@ -0,0 +1,42 @@
+package:
+    name: qutip
+    version: 3.1.0
+
+source:
+    fn: qutip-3.1.0.tar.gz
+    url: http://qutip.org/downloads/3.1.0/qutip-3.1.0.tar.gz
+
+build:
+    number: 0
+    script: $PYTHON setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - python >=2.7
+    - setuptools
```

Comment:
Looking at the code, it doesn't look like setuptools *is* actually a dependency (it is for running the tests though). For that reason, the ```--single-version-externally-managed --record=record.txt``` isn't needed (and indeed, doesn't work).